.github/actions/set_up_macos/action.yml

```diff
@@ -7,7 +7,7 @@ inputs:
     required: true
   xcode_version:
     description: The version of Xcode to use.
-    default: '14.3.1'
+    default: '15.0.1'
```

Comment:
Should we run the built-in frameworks update? it was last generated with Xcode 15.0.1, not sure if that gets caught by CI though.

Reply:
Then, we should be OK. Perhaps we need to tie the generation of the frameworks into CI or just generate as part of the build. 🤔

Reply:
I added #748 to track.